cli/src/cli_util.rs

```diff
@@ -3767,6 +3773,14 @@ fn format_template_aliases_hint(template_aliases: &TemplateAliasesMap) -> String
     hint
 }
 
+/// Determine, by its name, if a remote refers to the special local-only "git"
+/// remote that is used in the Git backend.
+///
+/// This function always returns false if the "git" feature is not enabled.
+pub fn is_special_git_remote(_remote: &str) -> bool {
```

Comment:
nit: `_remote` argument is used, and maybe this stub function is no longer needed?